File: mario.c
#include <cs50.h>
#include <stdio.h>

int get_height(void);
void pyramid(int h);

int main(void)
{
    int h = get_height();
    for(int i = 1; i <= h; i++)
    {
        string space = " ";
        int spacenumber = h-i;
        for(int sn = 1; sn <= h-i; sn++) /// sn for space number
        {
            printf("%s", space);
        }
        for(int ls = 1; ls <= i; ls++) /// ls for number of left sharp
        {
            printf("#");
        }
        printf("  ");
        for(int rs = 1; rs <= i; rs++) ///rs for number of right sharp
        {
            printf("#");
        }
        printf("\n");
    }
}

int get_height(void)
{
    int n;
    do
    {
       n = get_int("Height: ");
    }
    while (n<=0 || n>8);
    return n;
}

File: mario.py
import cs50


def main():
    pyramid()
    
    
def pyramid():
    while True:
        h = cs50.get_int("Height: ")
        if 0 < h < 9:
            break
    
    for i in range(1, h+1):
        spacenumber = h-i
        print(" " * spacenumber + "#" * i + "  " + "#" * i)


main()

# feedback from sytle50
# Looks good!
# But consider adding more comments!